content/source/docs/cloud/registry/design.html.md

```diff
@@ -23,15 +23,13 @@ Navigate to the modules list with the top-level "Modules" button, then click the
 
 ![Terraform Cloud screenshot: the design configuration button](./images/design-start.png)
 
-This brings you to the "Select Modules" page.
+The Select Modules page appears.
 
 ## Adding Modules
 
-![Terraform Cloud screenshot: the select modules page](./images/design-add-module.png)
```

Comment:
It feels a little weird not to have a screen shot here. Could we update or crop the existing one?

Reply:
okay fair! I added one back in :-)